src/app/tests/suites/commands/log/LogCommands.cpp

```diff
@@ -34,7 +34,7 @@ CHIP_ERROR LogCommands::UserPrompt(const char * identity,
     {
         std::string line;
         std::getline(std::cin, line);
-        if (line != value.expectedValue.Value().data())
+        if (line.compare(std::string(value.expectedValue.Value().data(), value.expectedValue.Value().size())))
```

Comment:
Can't this still use `!=`?  Why the change to `compare`?

Reply:
Updated.